File: PERL/Cap07/c07ex07.pl
# c07ex07.pl

#!/usr/bin/perl

use strict;
use warnings;

sub IGUAL {
  my ($A, $B) = @_;
  my $RESULT;
  if ($A == $B) {
    $RESULT = 0;
  } else {
    $RESULT = 1;
  }
  return $RESULT;
}

print "Informe o 1o. valor: ";
my $X = <STDIN>;
chomp $X;

print "Informe o 2o. valor: ";
my $Y = <STDIN>;
chomp $Y;

if (IGUAL($X, $Y) == 0) {
    print "Valores sao iguais\n";
} else {
    print "Valores sao diferentes\n";
}

print "\n";
print "Tecle <Enter> para encerrar... ";
<STDIN>;

File: PHP/Cap07/c07ex07.php
<?php
// c07ex07.php

function IGUAL($A, $B): bool {
  return $A == $B;
}

printf("Informe o 1o. valor: ");
$X = intval(trim(fgets(STDOUT)));
printf("Informe o 2o. valor: ");
$Y = intval(trim(fgets(STDOUT)));

if (IGUAL($X, $Y)) {
  printf("Valores sao iguais\n");
} else {
  printf("Valores sao diferentes\n");
}

printf("\n");
printf("Tecle <Enter> para encerrar... ");
trim(fgets(STDOUT));

?>
